api/v1beta1/disruption_types.go

```diff
@@ -44,7 +44,7 @@ type DisruptionSpec struct {
 	DryRun           bool                              `json:"dryRun,omitempty"`           // enable dry-run mode
 	OnInit           bool                              `json:"onInit,omitempty"`           // enable disruption on init
 	Unsafemode       *UnsafemodeSpec                   `json:"unsafeMode,omitempty"`       // unsafemode spec used to turn off safemode safety nets
-	StaticTargeting  bool                              `json:"staticTargeting,omitempty"`  // enable dynamic targeting and cluster observation
+	StaticTargeting  *bool                             `json:"staticTargeting,omitempty"`  // disable dynamic targeting and cluster observation
```

Comment:
Why did this have to become a pointer again? The default for a `bool` is already `false,` no?

Reply:
This PR aims to allow `true` to be a possible default value based on the chart
This value, if set, needs to not be overridden by an empty one

Reply:
I see. There is another option, I think? At the cost of some wordier unmarshaling, we can avoid making this be a pointer everywhere.

If we make the type of `StaticTargeting` not `bool`, but a custom type: `type StaticTargetingFlag bool`, we can add a custom Unmarshal for that type that uses the chart-defined default when empty.

Reply:
I tried a few things, hoping this would work; without a new dedicated struct type for the field -which would not really help- the UnmarshalJSON function is not called on an empty value 😞 
I'm afraid the pointer is unavoidable there

Reply:
Is it..... too late to change this field to "Targeting" with the options of "dynamic,static,empty", where empty uses the default? Representing three states by using a pointer to a bool feels wrong